e2e/ctl_v3_member_test.go

```diff
@@ -32,3 +41,72 @@ func ctlV3MemberList(cx ctlCtx) error {
 	}
 	return spawnWithExpects(cmdArgs, lines...)
 }
+
+type memberList struct {
```

Comment:
why a special struct? MemberListResponse can already be serialized/deserialized in json